src/routes/hardwareRouter.ts

```diff
@@ -7,6 +7,21 @@ export const hardwareRouter = (): Router => {
 
   const hardwareController = new HardwareController();
 
+  /**
+   * GET /hardware
+   */
+  router.get("/",
+    checkIsLoggedIn,
+    hardwareController.library);
+
+  /**
+   * GET /hardware/management
+   */
+  router.get("/management",
+    checkIsLoggedIn,
```

Comment:
We shouldn't just let anyone be able to access this page, we should restrict it to those who are volunteers.

Reply:
Good spot